comms/lighthouse/src/server.ts

```diff
@@ -60,6 +62,13 @@ async function main() {
   if (accessLogs) {
     app.use(morgan('combined'))
   }
+  if (validateAPI) {
+    OpenApiValidator.middleware({
+      apiSpec: COMMS_API,
+      // validateResponses: true, // false by default
```

Comment:
I'd never validate responses in code, only in tests

Reply:
```suggestion
      validateResponses: process.env.CI == 'true' // or isRunningTests()
```